README.md

```diff
@@ -139,18 +159,28 @@ $ sw-precache --root=dist --static-file-globs='dist/**/*.html'
 
 The `sw-precache` module exposes two methods: `generate` and `write`.
 
-#### generate(options, callback)
-`generate` takes in [options](#options), generates resulting service worker code as a string, and
-then invokes `callback(error, serviceWorkerString)`.
-In the 1.x releases of `sw-precache`, this was the default and only method exposed by the module.
+#### generate(options, callback) `generate` takes in [options](#options),
```

Comment:
This Markdown leads to what I assume is unintentional formatting:

![image](https://cloud.githubusercontent.com/assets/1749548/11076958/a5b384b0-87cb-11e5-9792-a0cd56a6e6f9.png)

In general, can you make sure you `View` the rendered Markdown using the GitHub web interface and check for problematic formatting? There are a few other places in this file that show similar issues (first lines of text being included in headings, for example). I can point them out one-by-one, but I think they'll stand out if you use the `View` option.